apps/astarte_pairing_api/lib/astarte_pairing_api_web/router.ex

```diff
@@ -22,6 +22,12 @@ defmodule Astarte.Pairing.APIWeb.Router do
   pipeline :api do
     plug :accepts, ["json"]
     plug Astarte.Pairing.APIWeb.Plug.LogRealm
+    plug Astarte.Pairing.APIWeb.Plug.AuthorizePath
+  end
+
+  pipeline :api_device do
+    plug :accepts, ["json"]
+    plug Astarte.Pairing.APIWeb.Plug.LogHwId
```

Comment:
The `Astarte.Pairing.APIWeb.Plug.LogRealm` plug is missing from this new pipeline, add it back